{"name": "test_mine[case5]", "status": "broken", "statusDetails": {"message": "uiautomator2.exceptions.UiObjectNotFoundError: -32001 Jsonrpc error: <androidx.test.uiautomator.UiObjectNotFoundException> data: UiSelector[TEXT=跳过], method: objInfo", "trace": "self = <test_mine.TestMine object at 0x00000175A4B9A6C8>\ncase = {'id': 'mine_006', '优先级': 'P1', '前置条件': '', '是否执行': 'yes', ...}\n\n    @pytest.mark.parametrize(\"case\", run_ed)\n    def test_mine(self, case):\n        # case_id = case[ExcelConfig.case_id]\n        # print(case_id)\n        # case_catalog = case[ExcelConfig.case_catalog]\n        # case_name = case[ExcelConfig.case_name]\n        # case_priority = case[ExcelConfig.case_priority]\n        # case_is_run = case[ExcelConfig.case_is_run]\n        # case_preconditions = case[ExcelConfig.case_preconditions]\n        # case_step = case[ExcelConfig.case_step]\n        case_step_node = case[ExcelConfig.case_step_node]\n        date1 = analysis(case_step_node)\n        # print(date1)\n        # case_expected_outcome = case[ExcelConfig.case_expected_outcome]\n        case_expected_node = case[ExcelConfig.case_expected_node]\n        data2 = expected_analysis(case_expected_node)\n        print(data2)\n>       d = ExecuteU2(\"851QFDSG22CDZ\", \"com.tencent.hobby\")\n\ntestcase\\test_mine.py:32: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nutils\\uiauto_util.py:27: in __init__\n    self.dv(text=\"跳过\").click()\nD:\\Program Files\\Python37\\lib\\site-packages\\uiautomator2\\_selector.py:151: in click\n    x, y = self.center(offset=offset)\nD:\\Program Files\\Python37\\lib\\site-packages\\uiautomator2\\_selector.py:178: in center\n    lx, ly, rx, ry = self.bounds()\nD:\\Program Files\\Python37\\lib\\site-packages\\uiautomator2\\_selector.py:165: in bounds\n    info = self.info\nD:\\Program Files\\Python37\\lib\\site-packages\\decorator.py:232: in fun\n    return caller(func, *(extras + args), **kw)\nD:\\Program Files\\Python37\\lib\\site-packages\\retry\\api.py:74: in retry_decorator\n    logger)\nD:\\Program Files\\Python37\\lib\\site-packages\\retry\\api.py:33: in __retry_internal\n    return f()\nD:\\Program Files\\Python37\\lib\\site-packages\\uiautomator2\\_selector.py:130: in info\n    return self.jsonrpc.objInfo(self.selector)\nD:\\Program Files\\Python37\\lib\\site-packages\\uiautomator2\\__init__.py:449: in __call__\n    http_timeout)\nD:\\Program Files\\Python37\\lib\\site-packages\\uiautomator2\\__init__.py:455: in _jsonrpc_retry_call\n    return self._jsonrpc_call(*args, **kwargs)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <uiautomator2.Device object at 0x00000175A4B9A148>, method = 'objInfo'\nparams = ({'mask': 1, 'childOrSibling': [], 'childOrSiblingSelector': [], 'text': '跳过'},)\nhttp_timeout = 60\n\n    def _jsonrpc_call(self, method, params=[], http_timeout=60):\n        \"\"\" jsonrpc2 call\n        Refs:\n            - http://www.jsonrpc.org/specification\n    \n        Raises:\n            出现的错误一般有2大类:\n                - JSONRPC服务端异常 ServerError\n                - 远程方法返回的错误 RequestError\n        \"\"\"\n        request_start = time.time()\n        data = {\n            \"jsonrpc\": \"2.0\",\n            \"id\": self._jsonrpc_id(method),\n            \"method\": method,\n            \"params\": params,\n        }\n        data = json.dumps(data)\n        res = self.http.post(\"/jsonrpc/0\",\n                             headers={\"Content-Type\": \"application/json\"},\n                             data=data,\n                             timeout=http_timeout)\n    \n        if res.status_code == 502:\n            raise GatewayError(\n                res, \"gateway error, time used %.1fs\" %\n                (time.time() - request_start))\n        if res.status_code == 410:  # http status gone: session broken\n            raise SessionBrokenError(\"app quit or crash\", res.text)\n        if res.status_code != 200:\n            raise BaseError(data, res.status_code, res.text,\n                            \"HTTP Return code is not 200\", res.text)\n        jsondata = res.json()\n        error = jsondata.get('error')\n        if not error:\n            return jsondata.get('result')\n    \n        err = JSONRPCError(error, method)\n    \n        def is_exception(err, exception_name):\n            return err.exception_name == exception_name or exception_name in err.message\n    \n        if isinstance(\n                err.data,\n                six.string_types) and 'UiAutomation not connected' in err.data:\n            err.__class__ = UiAutomationNotConnectedError\n        elif err.message:\n            if is_exception(err, 'uiautomator.UiObjectNotFoundException'):\n                err.__class__ = UiObjectNotFoundError\n            elif is_exception(\n                    err,\n                    'android.support.test.uiautomator.StaleObjectException'):\n                # StaleObjectException\n                # https://developer.android.com/reference/android/support/test/uiautomator/StaleObjectException.html\n                # A StaleObjectException exception is thrown when a UiObject2 is used after the underlying View has been destroyed.\n                # In this case, it is necessary to call findObject(BySelector) to obtain a new UiObject2 instance.\n                err.__class__ = StaleObjectExceptionError\n            elif is_exception(err, 'java.lang.NullObjectException'):\n                err.__class__ = NullObjectExceptionError\n            elif is_exception(err, 'java.lang.NullPointerException'):\n                err.__class__ = NullPointerExceptionError\n>       raise err\nE       uiautomator2.exceptions.UiObjectNotFoundError: -32001 Jsonrpc error: <androidx.test.uiautomator.UiObjectNotFoundException> data: UiSelector[TEXT=跳过], method: objInfo\n\nD:\\Program Files\\Python37\\lib\\site-packages\\uiautomator2\\__init__.py:527: UiObjectNotFoundError"}, "attachments": [{"name": "log", "source": "1ecb6660-ba14-4f5f-9631-dd5a377fe150-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'id': 'mine_006', '用例目录': '影响力', '用例名称': '影响力为0时', '优先级': 'P1', '是否执行': 'yes', '前置条件': '', '步骤': '1.嚯app启动》点击底部【我的】tab，我的画面，内容没被点赞过，内容没被评论数，内容没被转发过，内容没被收藏过，评论没被点赞过，评论没被点赞过，点击影响力', '步骤节点': '我的, 2202005276啊啊, 影响力', '预期结果': '1.影响力的总数为0，并且内容被点赞数为0，内容被评论数为0，内容被转发数为0，内容被收藏数为0，评论被点赞数为0，评论被回复数为0', '预期节点': ' ,  \\n我的影响力, 内容被点赞, 内容被评论, 内容被转发, 内容被收藏, 评论被点赞'}"}], "start": 1626765339069, "stop": 1626765372475, "uuid": "21860c27-e262-489c-870f-ce997d9701f6", "historyId": "38112308d51c6392a6ce8b9aace1f497", "testCaseId": "da5ae44c99fde797f954ed7865065f3e", "fullName": "testcase.test_mine.TestMine#test_mine", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_mine"}, {"name": "subSuite", "value": "TestMine"}, {"name": "host", "value": "v_linkaica-PC0"}, {"name": "thread", "value": "20816-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_mine"}]}